roles/kubernetes-apps/heapster/tasks/main.yml

```diff
@@ -0,0 +1,23 @@
+---
+- name: Heapster | Lay Down Heaspter Templates
+  template: src={{item.file}} dest={{kube_config_dir}}/{{item.file}}
```

Comment:
I guess 

```
- template:
    src: {{ item.file }}
    dest: { {item.file }}
```

is more readable. Could you please update to new format? Thank you in advance.